really not track inventory when inventory tracking is turned off

peterberkenbosch opened the pull request:
Work in progress:
- [ ] add specs
- [ ] search for other places that handle stock and do not use Spree::Config.track_inventory_levels correctly

peterberkenbosch committed:
when inventory levels are not tracked we still need to pack the packages.

peterberkenbosch committed:
no use moving stock when not tracking inventory

TeatroIO commented:
I've prepared a stage to preview changes. <a href='http://pero-ict-solutions-3-0-stable-pero.spree-spree-9ed724b77adec04c49c2.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://pero-ict-solutions-3-0-stable-pero.spree-spree-9ed724b77adec04c49c2.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

peterberkenbosch committed:
use 1.9 hash syntax

peterberkenbosch committed:
actually utilize the history option of friendlyId when visiting the product show page.
This will redirect with a 301 to the current url when the product#show is rendered with a legacy url

peterberkenbosch commented:
Closing this, since it's not from a specific feature branch. Will open a new one
